openwisp_controller/config/static/config/js/default_templates.js

```diff
@@ -46,10 +47,52 @@ django.jQuery(function ($) {
                 });
             });
         },
+        getRelevantTemplates = function () {
+            var orgID = $('#id_organization').val(),
+                backend = $('#id_config-0-backend').val();
+            if (!orgID || !backend) {
+                return;
+            }
+            // proceed only if an organization and a backend have been selected
+            if (orgID.length === 0 || backend.length === 0) {
+                return;
+            }
+            var url = '/admin/config/device/config/get-relevant-templates/';
```

Comment:
Can we try to avoid hardcoding the url here? 🤔 
Perhaps a variable entered from the template.
@nemesisdesign What do you think?

Reply:
Yes, we can use a variable as we are using for ``get_default_templates``. Should I add it?

Reply:
Yes, it's better because the URL will be automatically updated if we make changes in the python code. Thanks @atb00ker.